frontend/src/lib/candidate/components/logoutButton/LogoutButton.svelte

```diff
@@ -93,13 +89,18 @@ When set to false, the button variant is main.
   <div class="notification max-w-md text-center">
     <h2>{$t('candidateApp.logoutModal.title')}</h2>
     <br />
-    {#if remainingInfoAmount > 0}
+    {#if basicInfoQuestionsLeft && basicInfoQuestionsLeft > 0}
```

Comment:
It is sufficient to just check that `basicInfoQuestionsLeft > 0` because `undefined > 0` evaluates to `false`

Reply:
The code gives an error if basicInfoQuestionsLeft is not checked before comparing to numerical values